.swiftlint.yml

```diff
@@ -1,7 +1,9 @@
 included: # paths to include during linting. `--path` is ignored if present.
   - Source
```

Comment:
@deepesh-vasthimal-cko 

> @ehab-al-cko @alex-ioja-yang-cko We can also now enable / update CI / CD to check for linting.

I wouldn't rush to enforce linting on pipelines yet, its a bit of a waste of resources imo. We could instead review how we develop and see if its appropriate to get SwiftLint autocorrect inside the build scripts?

@harry-brown-cko have you done any discovery on doing the linting locally?